src/consensus.rs

```diff
@@ -79,8 +79,7 @@ pub struct Alpenglow<A: All2All, D: Disseminator, R: Network, T: Network> {
     all2all: Arc<A>,
     /// Block dissemination network protocol for shreds.
     disseminator: Arc<D>,
-    /// Block repair protocol.
-    repair: Arc<Repair<R>>,
+    _repair: std::marker::PhantomData<Repair<R>>,
```

Comment:
it's worth leaving a comment here explaining why we need phantom data.  It is generally a subtle reason that sometimes takes a while to wrap ones head around it.